packages/admin/cms-admin/package.json

```diff
@@ -132,7 +132,7 @@
         "react-dnd-html5-backend": "^16.0.0",
         "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0",
         "react-final-form": "^6.0.0",
-        "react-intl": "^5.0.0 || ^6.0.0",
+        "react-intl": "^5.0.0 || ^6.0.0 || ^7.0.0",
```

Comment:
should we drop support for 5 and/or 6?

Reply:
IMO there's no need to drop support since the APIs haven't changed between v5 and v7 and we're not using any of the newer features. Dropping support would result in devs having to upgrade TypeScript to v5, I'm not sure if we want to enforce this for react-intl when there's no actual reason to do so. 🤷🏼‍♂️

Reply:
Ok then we can keep thi